src/main/java/com/codeborne/selenide/webdriver/RemoteDriverFactory.java

```diff
@@ -52,6 +57,20 @@ DesiredCapabilities getDriverCapabilities(Proxy proxy) {
     if (headless) {
       capabilities.merge(getHeadlessCapabilities());
     }
+    if (!browserBinary.isEmpty()) {
+      log.info("Using browser binary: " + browserBinary);
```

Comment:
Consider extract this block to separate method like for `getHeadlessCapabilities()`